WooCommerce/Classes/ViewModels/Order Details/Receipts/ReceiptEligibilityUseCase.swift

```diff
@@ -42,6 +43,11 @@ final class ReceiptEligibilityUseCase: ReceiptEligibilityUseCaseProtocol {
         }
         stores.dispatch(action)
     }
+
+    func isEligibleSendingReceiptAfterPayment(onCompletion: @escaping (Bool) -> Void) {
+        // TODO: WooCommerce 9.5.0
+        onCompletion(false)
```

Comment:
Note: There's an issue with this functionality on the WooCommerce trunk (p91TBi-csN-p2#comment-13579) therefore I'm not implementing an eligibility handler. I recommend doing testing on my provided site.

Testing site https://lovely-aware-sturgeon.jurassic.ninja/ details: PdfdoF-5Qj-p2.

It means that the current changes should not affect production in any way.

Reply:
I think we should pass a feature flag here for now rather than passing `false` directly, just to be safe, then we can also modify it to perform a version check when 9.5 is available.